lib/AST/ASTImporter.cpp

```diff
@@ -2205,8 +2205,12 @@ Decl *ASTNodeImporter::VisitRecordDecl(RecordDecl *D) {
             // The record types structurally match, or the "from" translation
             // unit only had a forward declaration anyway; call it the same
             // function.
-            // FIXME: For C++, we should also merge methods here.
-            return Importer.MapImported(D, FoundDef);
+            Importer.MapImported(D, FoundDef);
```

Comment:
The enclosing condition is true if the class to be imported is either has a name and incomplete or it is anonymous and complete. I don't see how this is satisfied when the class to be imported has a name and is complete (the class in the test seems like having those properties).

Reply:
The class to be imported is complete and is not anonymous and the `FoundDef` is not anonymous too. The condition is true then.